pkg/controller/storageclusterinitialization/storageclusterinitialization_controller.go

```diff
@@ -472,7 +472,7 @@ func (r *ReconcileStorageClusterInitialization) newCephBlockPoolInstances(initDa
 				Namespace: initData.Namespace,
 			},
 			Spec: cephv1.PoolSpec{
-				FailureDomain: "host",
+				FailureDomain: initData.Spec.FailureDomain,
```

Comment:
I'm not sure. Since we're effectively just passing a variable from the InitData to this field, does it make sense to just have a test for setting values without a calculation?

Reply:
This was already technically being taken care of by the unit tests. I added an explicit configuration of `failureDomain` to the scinit objects just to be sure.